### DISCUSSION TITLE
[Issue]: API, how to set playback progress?

### DISCUSSION THREAD

**Discussion by drgroot** [jellyfin/jellyfin#7259] [Q&A]
### Please describe your bug

The api allows us to set a user's item as played or unplayed. How can we set the playback for another user at a particular position tick for a given item? I cannot seem to find it in the API.

### Jellyfin Version

10.7.7

### Environment

```markdown
- Component: API
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nielsvanvelzen** [comment]
You can find the API documentation for the latest release at https://api.jellyfin.org.

I think you want to use the [OnPlaybackProgress](https://api.jellyfin.org/#operation/OnPlaybackProgress) endpoint. The URL contains the user and item id like so: `/Users/{userId}/PlayingItems/{itemId}/Progress`.

**Comment by drgroot** [comment]
the doesnt seem to work,

```
post: https://<URL>/Users/<user_id>/PlayingItems/<item_id>/Progress
params: { positionTicks: 7000 }
```

This returns with `Request Failed with status code 400. Responded with "Error processing request."`

**Comment by drgroot** [comment]
following these endpoints doesnt work
https://emby.media/community/index.php?/topic/75926-what-endpoint-to-update-a-users-watched-position/

It seems that there is no way in Jellyfin to do this. Using the 'PlayingItems' assumes there is an active session. In my case, there is no active session.

**Comment by luigi311** [comment]
Has anyone found out how to do this? I was able to use a delete request with the following /Users/{user_id}/PlayingItems/{jellyfin_video_id}?positionTicks={tick_value} but that only works with a user token and not the api key which means i cant use it on my application.

**Reply by CoolnsX** [reply]
THANK YOU!!, by the way, you can update it by encapsulating token key in authorization like this --

```
Authorization: MediaBrowser Token="{JF_TOKEN}"
```

please mind the double quotes as they are important

also the API endpoint is

```sh
curl -X 'DELETE' \
  'https://jellyfin.example.com/PlayingItems/475ee57f-b019-7b01-e183-fdaeadd2c0da?positionTicks=1818650000' \ # dummy itemID
  -H 'accept: */*' \
  -H 'Authorization: MediaBrowser Token="0ceb4eb2a9cd4ce2aba5164740a761bc"' # dummy token
```
